google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/wrappers/Filters.java

```diff
@@ -151,6 +149,15 @@ public LimitFilter limit() {
   }
 
   // Miscellaneous filters without a clear target.
+  /**
+   * Wraps protobuf representation of a filter.
+   *
+   * <p>For advanced use only.
+   */
+  public Filter raw(RowFilter rowFilter) {
```

Comment:
This is going "raw" to "nice" right? In that case, maybe call this "fromRaw" instead? The name "raw" seems to imply "raw representation of the argument".

Reply:
You are right, but maybe `fromProto` is better? If you agree, please merge :)